### PR TITLE
Add composition

### PR DESCRIPTION
**Pull request by jvshields**
Adding structure for composition class

### CODE REVIEW

File: stardis/io/model/marcs.py [hunk 1]
```diff
@@ -30,6 +31,21 @@ def to_geometry(self):
         r = self.data.depth.values * u.cm
         return Radial1DGeometry(r)
 
+    # def to_composition(self):
```
Comment (sonachitchyan) [inline stardis/io/model/marcs.py:34]: add something like "this will be used later" ?

File: stardis/model/base.py [hunk 2]
```diff
@@ -110,4 +115,6 @@ def read_marcs_to_fv(fpath, atom_data, final_atomic_number=30):
         marcs_abundances_all[i] = marcs_abundances["mass_abundance"]
     marcs_abundances_all = marcs_abundances_all[:final_atomic_number]
 
-    return StellarModel(marcs_model_fv, marcs_abundances_all, boundary_temps, geometry)
+    return StellarModel(
```
Comment (sonachitchyan) [inline stardis/model/base.py:118]: this class is getting resturctured later right?

Comment (jvshields) [reply stardis/model/base.py:118]: Yes. The current plan is to add the different attributes to carry along the information in the way that we actually want, and then switch over and get rid of the old redundant attributes later once the code can run without them (specifically marcs_model_fv, marcs_abundanes_all, and boundary_temps).

File: stardis/io/model/marcs.py (hunk 1 above)
Comment (jvshields) [reply stardis/io/model/marcs.py:34]: This is a reasonable suggestion that I don't disagree with on principle, but as soon as this gets merged I'll be uncommenting and hooking this method up with a branch that prepares the StellarModel class from a marcs model. I'd rather not go back and deal with the merge conflicts on that branch to add this comment that will be short-lived.

**Review by sonachitchyan — APPROVED** [review]

**Review by DeerWhale — APPROVED** [review]
The structure of having composition as an attribute looks good.

**Pull request merged**